cicecore/cicedynB/infrastructure/comm/mpi/ice_timers.F90

```diff
@@ -62,8 +65,18 @@ module ice_timers
       timer_bgc,              &! biogeochemistry
       timer_forcing,          &! forcing
       timer_evp_1d,           &! timer only loop
-      timer_evp_2d             ! timer including conversion 1d/2d
-!      timer_tmp               ! for temporary timings
+      timer_evp_2d,           &! timer including conversion 1d/2d
+      timer_updstate           ! update state
+!      timer_updstate,         &! update state
+!      timer_tmp1,             &! for temporary timings
+!      timer_tmp2,             &! for temporary timings
+!      timer_tmp3,             &! for temporary timings
+!      timer_tmp4,             &! for temporary timings
+!      timer_tmp5,             &! for temporary timings
```

Comment:
are you planning to clean these tmp timers up later?

Reply:
I go back and forth on this.  I was thinking I'd just leave them there but commented out.  I find when I'm debugging any performance thing, I end up implementing the same temporary timers and dropping them in where I need a finer detail look.  Then removing them at the end.  It might be nice to just have them readily available anytime anyone needs to use them.  I can remove them if that's preferred.

Reply:
I'm ambivalent.  Sometimes it's nice to have commented-out code sitting around for ease of use.  Comments/opinions from others are welcome!